doc/source/ray-contribute/getting-involved.rst

```diff
@@ -51,7 +51,7 @@ ones labeled `"good first issue" <https://github.com/ray-project/ray/issues?utf8
 Setting up your development environment
 ---------------------------------------
 
-To edit the Ray source code, you'll want to checkout the repository and also build Ray from source. Follow :ref:`these instructions for building <building-ray>` a local copy of Ray to easily make changes.
+To edit the Ray source code, you'll want to fork the repository, clone it and build Ray from source. Follow :ref:`these instructions for building <building-ray>` a local copy of Ray to easily make changes.
```

Comment:
```suggestion
To edit the Ray source code, fork the repository, clone it, and build Ray from source. Follow :ref:`these instructions for building <building-ray>` a local copy of Ray to easily make changes.
```

Reply:
Thanks for the suggestion. will the update it